### PR TITLE
Add editorconfig

### PR DESCRIPTION
**Pull request by JoseCage**
Added editorconfig

### CODE REVIEW

**Comment by sdmg15** [comment]
OK thanks for this I'll move to PSR-2.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```
Comment (sdmg15) [inline .editorconfig:5]: The indent size should be 4 according to PSR-2

Comment (JoseCage) [reply .editorconfig:5]: 👍
You just need to edit `.editorconfig` file and add `indent_size = 4`

**Comment by JoseCage** [comment]
Right @sdmg15 

 Watching for your updates..

> OK thanks for this I'll move to PSR-2.

**Pull request merged**